tests/test_intervals.py

```diff
@@ -349,3 +349,615 @@ def test_points_per_interval_slicer(test_data, ppi_params_and_ref):
         np.testing.assert_array_equal(my_intervals[i], ref_intervals[i])
     for i in range(len(my_boundaries)):
         np.testing.assert_almost_equal(my_boundaries[i], ref_boundaries[i])
+        
+
+def test_points_per_interval_slicer_npoints2(test_data):
+    """
+    Function to compare characterisics of reference interval generated by
+    PointsPerIntervalSlicer and manually generated interval..
+
+    Parameters
+    ----------
+    test_data : numpy array
+        Data to test the interval slicer.
+ 
+    Returns
+    -------
+    None.
+
+    """
+    
+    # n_points= 2
```

Comment:
What does this comment mean? Please either describe it clearly using a full sentence or delete it.